pkg/virt-handler/mount-manager/mount.go

```diff
@@ -0,0 +1,109 @@
+package mount
+
+import (
+	"fmt"
+	"time"
+
+	"k8s.io/apimachinery/pkg/types"
+
+	v1 "kubevirt.io/api/core/v1"
+
+	containerdisk "kubevirt.io/kubevirt/pkg/container-disk"
+	virtconfig "kubevirt.io/kubevirt/pkg/virt-config"
+	container_disk "kubevirt.io/kubevirt/pkg/virt-handler/container-disk"
+	hotplug_volume "kubevirt.io/kubevirt/pkg/virt-handler/hotplug-disk"
+	"kubevirt.io/kubevirt/pkg/virt-handler/isolation"
+)
+
+// MountInfo wraps all the mount information
+type MountInfo struct {
+	containerDisksInfo map[string]*containerdisk.DiskInfo
+}
+
+func (info *MountInfo) GetContainerDisksInfo() map[string]*containerdisk.DiskInfo {
+	return info.containerDisksInfo
+}
+
+// MountManager handles all the mount operations required by KubeVirt
+type MountManager interface {
+	Mount(vmi *v1.VirtualMachineInstance) (MountInfo, error)
```

Comment:
Not sure. I won't put it in the mount because there is the queuing part. I could merge but it makes sense to me to have it in a separate function.  In one of the first versions I have called this `MountReady` but then we decided to keep it more specific from @vasiliy-ul [comment](https://github.com/kubevirt/kubevirt/pull/8317#discussion_r1094268436)